app/components/windows/GameOverlayEventFeed.m.less

```diff
@@ -0,0 +1,7 @@
+@import "../../styles/index.less";
```

Comment:
this compiles to nothing so not a big deal but just FYI you dont need to import anything to use css variables

Reply:
I didn't know that. I'll stop doing this everywhere